web/versioned_docs/version-0.14.0/advanced/deployment/manually.md

```diff
@@ -209,7 +209,7 @@ flyctl deploy --remote-only --config ../../fly.toml
 
 This will build and deploy the backend of your Wasp app on Fly.io to `https://<app-name>.fly.dev` 🤘🎸
 
-Now, if you haven't, you can deploy your client and add the client URL by running `flyctl secrets set WASP_WEB_CLIENT_URL=<url_of_deployed_client>`. We suggest using [Netlify](#netlify) for your client, but you can use any static hosting provider.
+Now, if you haven't, you can deploy your client and add the client URL by running `flyctl secrets set WASP_WEB_CLIENT_URL=<url_of_deployed_client>`. We suggest using [Netlify](#netlify-client) for your client, but you can use any static hosting provider.
```

Comment:
Didn't you just ensure in 0.12 that the links are just `(#netlify)` etc? Should we stick to one way of doing it. 

Non-blocking, these are old docs so they should be kinda "frozen" anyway.